Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -154,6 +184,32 @@ def fetch_indicators_command(client: Client, indicator_type: str, **kwargs) -> U
     return indicators
 
 
+def extract_all_fields_from_indicator(indicator, indicator_key):
```

Comment:
Why not to join the values here? you may override keys here

Reply:
Fixed